src/components/Alert/AlertLink.tsx

```diff
@@ -12,7 +12,11 @@ export const AlertLink = ({
   isExternal
 }: AlertLinkProperties): JSX.Element => (
   <li className='m-list_item'>
-    <a className='m-list_link' href={href}>
+    <a
+      className='m-list_link'
+      href={href}
+      aria-label={`${label || ''} external link}`}
```

Comment:
Actually, this line will add "external link" to all alert links, even if they aren't external?
<img width="1244" alt="Screenshot 2024-05-30 at 6 19 04 PM" src="https://github.com/cfpb/design-system-react/assets/19983248/7411f670-1890-4fc2-9d75-63eae24cd50e">